src/ResourceManager/Network/ChangeLog.md

```diff
@@ -34,6 +34,16 @@
 * Add support for DDoS protection plan resource
 * Introduced multiple breaking changes
     - Please refer to the migration guide for more information
+* Updated below commands for feature: Point to Site IPsec custom policy set/remove on Brooklyn Gateway.
```

Comment:
@Nilambari please move the snippet you are adding to below the `## Current Release` header